.gitignore

```diff
@@ -31,3 +31,5 @@ updates_key.pem
 test/testdata
 .tox
 youtube-dl.zsh
+.idea
```

Comment:
This is probably specific to your editor, you [should add them to your global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) instead.